src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -583,6 +583,8 @@ private static void Initialize()
 
                     // Ensure that we allow the unhandled exception to kill the process.
                     // unhandled Node global exceptions should never be swallowed.
+                    // Delay to allow logging to complete before throwing exception.
+                    Task.Delay(500).Wait();
```

Comment:
The above event multicast delegate is a synchronous call, and our ScriptHost event delegate synchronously logs then flushes the logs. So why is a delay needed?